packages/avocode-email-tagsinput/test/e2e/basic.spec.js

```diff
@@ -0,0 +1,116 @@
+import puppeteer from 'puppeteer'
```

Comment:
Please rename this file to be `packages/avocode-email-tagsinput/test/e2e/avocode-email-tagsinput.spec.js` since you're directly testing that single component.

I think we haven't got to the point where we should separate spec files based on testing scenarios. So let's keep it in one file for now.